packages/react/src/components/List/List.tsx

```diff
@@ -70,122 +76,143 @@ export interface ListProps extends UIComponentProps, ChildrenComponentProps {
   wrap?: (children: ReactChildren) => React.ReactNode
 }
 
-export interface ListState {
-  selectedIndex?: number
-}
+// List props that are passed to each individual Item props
+const itemProps = [
+  'debug',
+  'selectable',
+  'navigable',
+  'truncateContent',
+  'truncateHeader',
+  'variables',
+]
+
+const List: React.FC<WithAsProp<ListProps>> &
+  FluentComponentStaticProps<ListProps> & {
+    Item: typeof ListItem
+  } = props => {
+  const context: ProviderContextPrepared = React.useContext(ThemeContext)
+  const { setStart, setEnd } = useTelemetry(List.displayName, context.telemetry)
+  setStart()
+
+  const {
+    accessibility,
+    as,
+    children,
+    className,
+    debug,
+    defaultSelectedIndex,
+    design,
+    horizontal,
+    navigable,
+    items,
+    selectable,
+    selectedIndex,
+    styles,
+    variables,
+    wrap,
+  } = props
+  setStart()
+
+  const { state, actions } = useStateManager(createListManager, {
+    mapPropsToInitialState: () => ({ selectedIndex: defaultSelectedIndex }),
+    mapPropsToState: () => ({ selectedIndex }),
+  })
+  const getA11Props = useAccessibility(accessibility, {
+    debugName: List.displayName,
+    mapPropsToBehavior: () => ({
+      horizontal,
+      navigable,
+      selectable,
+    }),
+    rtl: context.rtl,
+  })
+  const { classes } = useStyles(List.displayName, {
+    className: List.className,
+    mapPropsToStyles: () => ({ isListTag: as === 'ol' || as === 'ul', debug, horizontal }),
```

Comment:
`isListTag` - love it!
`selectable` - oh man I feel sorry if someone needs it...

Reply:
> selectable - oh man I feel sorry if someone needs it...

Let's see, what I can say 🤔

Reply:
It would be nice once we will extract all design-related props from the components :)